Makefile

```diff
@@ -290,14 +295,14 @@ lint: export TARGET := default
 lint: ##@test Run code style checks
 	sh scripts/lint-re-frame-in-quo-components.sh && \
 	clj-kondo --config .clj-kondo/config.edn --cache false --lint src && \
-	ALL_CLOJURE_FILE=$$(git ls-files | grep -e \.clj$$ -e \.cljs$$ -e \.cljc$$ -e \.edn$$) && \
-	zprint '{:search-config? true}' -sfc $$ALL_CLOJURE_FILE
+	ALL_CLOJURE_FILES=$(call all_clojure_files) && \
+	zprint '{:search-config? true}' -sfc $$ALL_CLOJURE_FILES
 
 lint-fix: export TARGET := default
 lint-fix: ##@test Run code style checks and fix issues
-	ALL_CLOJURE_FILE=$$(git ls-files | grep -e \.clj$$ -e \.cljs$$ -e \.cljc$$ -e \.edn$$) && \
-	zprint '{:search-config? true}' -sw $$ALL_CLOJURE_FILE && \
-	zprint '{:search-config? true}' -sw $$ALL_CLOJURE_FILE
+	ALL_CLOJURE_FILES=$(call all_clojure_files) && \
+	zprint '{:search-config? true}' -sw $$ALL_CLOJURE_FILES && \
+	zprint '{:search-config? true}' -sw $$ALL_CLOJURE_FILES
```

Comment:
Why is this command being called twice? Is this some sort of hack-fix?

Reply:
Yes, it's a temporary workaround while the maintainer of zprint is working on a fix. @yqrashawn already reported the bug.

Reply:
I see, then a `TODO`/`FIXME` comment explaining that is usually in order.

Reply:
Reference:

- https://github.com/kkinnear/zprint/issues/271